programs/bpf/c/src/invoked/invoked.c

```diff
@@ -36,7 +36,7 @@ extern uint64_t entrypoint(const uint8_t *input) {
     sol_assert(accounts[ARGUMENT_INDEX].data_len == 100);
     sol_assert(accounts[ARGUMENT_INDEX].is_signer);
     sol_assert(accounts[ARGUMENT_INDEX].is_writable);
-    sol_assert(accounts[ARGUMENT_INDEX].rent_epoch == 1);
+    sol_assert(accounts[ARGUMENT_INDEX].rent_epoch == 0);
```

Comment:
@jackcmay Like [the RPC land](https://github.com/solana-labs/solana/pull/11349/files#r469692568), this pr changes slightly changes [the semantics of `rent_epoch`](https://github.com/solana-labs/solana/pull/11349/files#r469691300), which is exposed to programs as well. Are you aware of any programs strictly relying on the current semantics of `rent_epoch`?